single_cell_analysis/scripts/util/vis.R

```diff
@@ -143,3 +143,92 @@ Z_dm_perf_vis <- function(perf.table){
   
   return(p)
 }
+
+learning_curve_plot <- function(filename, input.dir){
+  # plot training/validation loss
+  # Args:
+  #  lc.table: data frame contains training and validation loss for each epoch
+  #  filename: file name of lc.table
+  # Returns:
+  #  ggplot object that plot learning curve
+  
+  lc.table <- read.table(file.path(input.dir, filename), sep = "\t", header = TRUE)
```

Comment:
check out [`readr::read_tsv`](https://readr.tidyverse.org/reference/read_delim.html). Much faster than `read.table` especially when the data is large.